docs/index.md

```diff
@@ -70,6 +70,8 @@ the Python ecosystem, **napari** can be easily coupled to leading machine
 learning and image analysis tools (e.g. `scikit-image`, `scikit-learn`,
 `TensorFlow`, `PyTorch`), enabling more user-friendly automated analysis.
 
+![image](./images/screenshot-add-image.png)
```

Comment:
```suggestion
![image](./images/multichannel_cells.png)
```
maybe use this?

Reply:
yep, could do this! If we go with this, I will update the screenshot there too - do you know anything about the LFS situation with the docs?

Reply:
The best person to ask is @jni but if I good remember now we should push every image using LFS until a separate repo for docs will be created.